pkg/policy/cell/cell.go

```diff
@@ -26,9 +25,8 @@ var Cell = cell.Module(
 	"policy",
 	"Contains policy rules",
 
-	cell.Provide(newPolicyRepo),
-	cell.Provide(newPolicyUpdater),
-	cell.Provide(newPolicyImporter),
+	cell.Provide(newPolicyRepo, newPolicyQueue),
```

Comment:
nit: the policyQueue seems somewhat internal, can you get away with `ProvidePrivate`? One way to do so would be to provide the `PolicyUpdater` via an adaptor here; i.e. `cell.Provide(func (q *policyQueue) PolicyUpdater {return q})`